### ISSUE TITLE
Functions Reference docs bug

### ISSUE THREAD

**Issue by ryan-richt** [stan-dev/docs#24]
#### Summary:
qr_R doc text has an error

#### Description:
Currently it reads:

```
matrix qr_R(matrix A)
The upper trapezoidal matrix in the fat QR decomposition of A, 
which implies that the resulting matrix will be square with the same number of rows as A
```

I think that the last part should say "...the resulting matrix will be **rectangular with the same dimensions as A**"

#### Additional Information:
If I run the program:
```
data {
  int n;
  int p;
  matrix[n,p] ss_v1;
}
model {
}
generated quantities {
  matrix[n,p] foo;
  {
    foo = qr_R(ss_v1);
  }
}
```
with Data:
```
  val matrix = Seq[Seq[Double]](Seq(17, 9, 13), Seq(9, 22, 31), Seq(13, 31, 50), Seq(1, 4, 9), Seq(8, 20, 36), Seq(21, 48, 81))
```

I get back the rectangular matrix:
```
Vector(
  Vector(32.3265, 59.5797, 97.3815), 
  Vector(-0.0, 26.3868, 45.1378), 
  Vector(-0.0, -0.0, 6.88628), 
  Vector(0.0, 0.0, 0.0), 
  Vector(0.0, 0.0, 0.0), 
  Vector(0.0, 0.0, 0.0)
)
```

Which seems to confirm the change. If I change the dimensions of the generated quantity to be N x N or P x P, it does not compile in CmdStan.

#### Current Version:
v2.18.0

**Comment by ryan-richt** [comment]
Also I wanted to follow the contributing guidelines but it doesn't seem that such master a docs tracking issue exists right now?

**Comment by bob-carpenter** [comment]
Thanks!  @mitzimorris has been reorganizing all the doc stuff, so I hope she can help.

**Comment by mitzimorris** [comment]
fixed via #15
